courses/free-courses-en.md

```diff
@@ -255,6 +255,7 @@
 * [SQL Foundations](https://www.udemy.com/course/sql-essentials-for-beginners/) - Sagar Uppuluri (Udemy)
 * [SQL Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWr_6xTfF2FrIw-NAOo3iWMy)
 * [SQL Tutorial - Full Database Course for Beginners](https://www.youtube.com/watch?v=HXV3zeQKqGY) - Mike Dane (freeCodeCamp)
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite/index.htm)
```

Comment:
```suggestion
* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite/)
```

The right category is `books/free-programming-books-langs.md`.

Reply:
It isn't a book.

Reply:
It's a text book.

From [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md):
>Books vs. Courses
>
>Sometimes these can be hard to distinguish!
>
>Courses often have associated textbooks, which we would list in our books lists. Courses have lectures, exercises, tests, notes or other didactic aids. A single lecture or video by itself is not a course. A powerpoint is not a course.

There are tutorialspoint link both in the courses section and in the books section so maybe it's better to wait for indications from some other more experienced staffer.

Reply:
Maybe the past opened discussion #4573 could help, it talks about `TutorialsPoint`